src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -383,6 +348,65 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                 syntax.AccessorList, syntax.GetExpressionBodySyntax(), syntax, diagnostics);
         }
 
+        private static void GetAcessorDeclarations(BasePropertyDeclarationSyntax syntax, DiagnosticBag diagnostics, 
+                                                   out bool notRegularProperty, out bool hasAccessorList, 
+                                                   out AccessorDeclarationSyntax getSyntax, out AccessorDeclarationSyntax setSyntax)
+        {
+            notRegularProperty = true;
```

Comment:
Nit: consider flipping this to false by default, setting it to true only if `hasAccessorList` and removing the `else` below.